kornia/geometry/linalg.py

```diff
@@ -200,5 +197,31 @@ def transform_points(trans_01: torch.Tensor, points_1: torch.Tensor) -> torch.Te
     points_0 = points_0.reshape(shape_inp)
     return points_0
 
+
+def points_lines_distances(pts: torch.Tensor, lines: torch.Tensor):
+    r"""Return the distance from points to lines.
+
+    Args:
+       pts: (possibly homogeneous) points :math:`(*, N, 2 or 3)`.
+       lines: lines coefficients (a, b, c) :math:`(*, N, 3)`, where :math:`ax + by + c = 0`.
+
+    Returns:
+        the computed distance with shape :math:`(*, N)`.
+    """
+    KORNIA_CHECK_IS_TENSOR(pts)
+    KORNIA_CHECK_IS_TENSOR(lines)
+
+    if not pts.shape[-1] in (2, 3):
+        raise ValueError(f"pts must be a (*, 2 or 3) tensor. Got {pts.shape}")
+
+    if not lines.shape[-1] == 3:
+        raise ValueError(f"lines must be a (*, 3) tensor. Got {lines.shape}")
+
+    numerator = torch.abs(lines[..., 0] * pts[..., 0] + lines[..., 1] * pts[..., 1] + lines[..., 2])
+    denominator = torch.linalg.norm(lines[..., :2], dim=-1)
```

Comment:
```suggestion
    denominator = lines[..., :2].norm(dim=-1)
```
i'd like more to remove `dim` kwarg to make it more generic in future with numpy or jax though